story01 (#1)
alem-platform/bootcamp-js: story02/filter-force/README.md

# Filter Force

> Your mission in this problem is to filter through the endless sea of elements and separate the worthy from the unworthy. Using the mighty `filter` array method, you must pass each element through a test and return a new array with only those that succeed!

Reviewer: ```suggestion
Your mission in this problem is to filter through the elements and separate the worthy from the unworthy.

### Resources: 
(filter)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter]
```

### Instructions:

Create a function called `filterForce` that takes two arguments:

> 1. An array: The sea of elements you’ll be filtering through.

Reviewer: ```suggestion
1. An array: Elements you’ll be filtering through.
```
